CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.18.0 - 2021-07-21
+### Added
+- Postgres Flexible Servers (Preview): CRUD support for Databases, Functions and Stored Procedures
```

Comment:
I wouldn't use an acronym like `CRUD` - not everyone remembers what those stand for